<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupEditor</class>
 <widget class="QWizard" name="BackupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::CancelButtonOnLeft</set>
  </property>
  <widget class="QWizardPage" name="BackupNamePage">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Backup Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="BackupNameEdit">
      <property name="text">
       <string>NewBackup1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Comment</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="CommentEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="SrcEditorPage">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Files for backup</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="rotateButton">
        <property name="toolTip">
         <string>Orientation</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="backupeditor.qrc">
          <normaloff>:/Orientation-2-2-icon.png</normaloff>:/Orientation-2-2-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="FrameSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QFrame" name="FirstFrame">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Inclusion Rules</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="SrcDescTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="SecondFrame">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>File System</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="TraversableTreeView" name="FileSystemTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="StoragesPage">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Storages</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="StoragesGroupBox">
      <property name="title">
       <string>Override backup-list settings</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="StorageFrame" name="StorageWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="ToolChainPage">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Toolchain</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ToolChainGroupBox">
      <property name="title">
       <string>Override backup-list settings</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="ToolChainFrame" name="ToolChainWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="SchemePage">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Scheme</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="SchemeGroupBox">
      <property name="title">
       <string>Override backup-list settings</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="SchemeFrame" name="SchemeWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="BackupNowCheckBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Create first backup now</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="AddWildcardDialogAction">
   <property name="text">
    <string>Add wildcard filter...</string>
   </property>
   <property name="toolTip">
    <string>Add new wildcard filter</string>
   </property>
  </action>
  <action name="AddRegExpDialogAction">
   <property name="text">
    <string>Add regexp filter...</string>
   </property>
   <property name="toolTip">
    <string>Add new regexp filter</string>
   </property>
  </action>
  <action name="DepthAction">
   <property name="text">
    <string>Set depth limit...</string>
   </property>
   <property name="toolTip">
    <string>Set depth limit</string>
   </property>
  </action>
  <action name="RemoveSelectedAction">
   <property name="text">
    <string>Remove selected</string>
   </property>
   <property name="toolTip">
    <string>Remove selected paths and filters</string>
   </property>
  </action>
  <action name="SetWildcardDialogAction">
   <property name="text">
    <string>Edit selected wildcard filters...</string>
   </property>
   <property name="toolTip">
    <string>Edit selected wildcard filters</string>
   </property>
  </action>
  <action name="SetRegExpDialogAction">
   <property name="text">
    <string>Edit selected regexp filters...</string>
   </property>
   <property name="toolTip">
    <string>Edit selected regexp filters</string>
   </property>
  </action>
  <action name="AddInclusionSrcPathAction">
   <property name="text">
    <string>Include path</string>
   </property>
   <property name="toolTip">
    <string>Add entry to backup inclusion rules</string>
   </property>
  </action>
  <action name="AddExclusionSrcPathAction">
   <property name="text">
    <string>Exclude path</string>
   </property>
   <property name="toolTip">
    <string>Add entry as exclusion rule</string>
   </property>
  </action>
  <action name="RemoveSrcPathAction">
   <property name="text">
    <string>Remove rule</string>
   </property>
   <property name="toolTip">
    <string>Remove rule for selected entry</string>
   </property>
  </action>
  <action name="TraverseAction">
   <property name="text">
    <string>Traverse</string>
   </property>
   <property name="toolTip">
    <string>Check entries included to backup</string>
   </property>
  </action>
  <action name="AddWildcardDialogFSAction">
   <property name="text">
    <string>Add wildcard filter...</string>
   </property>
   <property name="toolTip">
    <string>Add new wildcard filter</string>
   </property>
  </action>
  <action name="AddRegExpDialogFSAction">
   <property name="text">
    <string>Add regexp filter...</string>
   </property>
   <property name="toolTip">
    <string>Add new regexp filter</string>
   </property>
  </action>
  <action name="DepthFSAction">
   <property name="text">
    <string>Set depth limit...</string>
   </property>
   <property name="toolTip">
    <string>Set depth limit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StorageFrame</class>
   <extends>QWidget</extends>
   <header>StorageFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolChainFrame</class>
   <extends>QWidget</extends>
   <header>ToolChainFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SchemeFrame</class>
   <extends>QWidget</extends>
   <header>SchemeFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TraversableTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">TraversableTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="backupeditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ToolChainGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackupEditor</receiver>
   <slot>OverrideToolchain(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StoragesGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackupEditor</receiver>
   <slot>OverrideStorages(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SchemeGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackupEditor</receiver>
   <slot>OverrideScheme(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OverrideToolchain(bool)</slot>
  <slot>OverrideStorages(bool)</slot>
  <slot>OverrideScheme(bool)</slot>
  <slot>ValidateBackupName()</slot>
 </slots>
</ui>
